internal/e2e/issue_variant_query_test.go

```diff
@@ -312,8 +349,12 @@ var _ = Describe("Updating issueVariant via API", Label("e2e", "IssueVariants"),
 				issueVariant.SecondaryName = "SecretIssueVariant"
 
 				req.Var("id", fmt.Sprintf("%d", issueVariant.Id))
-				req.Var("input", map[string]string{
+				req.Var("input", map[string]interface{}{
 					"secondaryName": issueVariant.SecondaryName,
+					"severity": model.SeverityInput{
+						Rating: &model.AllSeverityValues[0],
+						Vector: &issueVariant.Severity.Cvss.Vector,
```

Comment:
Do we check if rating and vector match?

Reply:
I added logic now to check if rating & vector don't match when they are both passed. If they match it works like normal, if not the issue variant creation errors out.